app/__init__.py

```diff
@@ -3,5 +3,6 @@
 
```

Comment:
Consider adding a couple more commits, for instance one for the `Planet` class and one after creating each route